egs2/commonvoice/asr1/README.md

```diff
@@ -277,3 +277,33 @@ For more details on how use fusion of front-ends, you can refer to this [wiki](h
 |decode_valid_dev_kabdecode_lm_eval.loss.best_asr_model_valid.loss.best|17766|356649|83.9|10.5|5.6|2.9|18.9|76.1|
 |decode_valid_test_kabdecode_lm_eval.loss.best_asr_model_valid.loss.best|18221|350154|84.1|10.4|5.5|2.9|18.8|76.1|
 
+# RESULTS
+## Environments
+- date: `Thu Sep  1 16:16:04 UTC 2022`
+- python version: `3.9.12 (main, Jun  1 2022, 11:38:51)  [GCC 7.5.0]`
+- espnet version: `espnet 202207`
+- pytorch version: `pytorch 1.12.1+cu102`
+- Git hash: `13db69d3befc3c82a5ff5a11e28bf79d5030603f`
+  - Commit date: `Mon Aug 29 13:44:35 2022 +0000`
+
+## asr_train_asr_conformer5_raw_zh_TW_bpe2542_sp_lowlr
```

Comment:
@sw005320 BPE size is set to `2500` for `zh` in run script. It may be better to add conditions for `zh_CN`, `zh_TW` and `zh_HK` if the sizes are different?

Also, what does `lowlr` tag means here? Maybe print the value used in the tag `lr[X]` instead? If `train_conformer5.yaml` was further modified, it should be highlighted

Reply:
> @sw005320 BPE size is set to `2500` for `zh` in run script. It may be better to add conditions for `zh_CN`, `zh_TW` and `zh_HK` if the sizes are different?

Year, I think so.
@slSeanWU, what is the character vocabulary size of `zh_TW`?

Reply:
How about I rename the tag and put the modified config on HF?

Reply:
@sw005320 I tried 150 at first, then an error was thrown, telling me it should be at least 2542.

Reply:
so I guess it's 2542 minus some special tokens

Reply:
got it.